Sources/ComposableCoreLocation/Live.swift

```diff
@@ -3,25 +3,30 @@ import ComposableArchitecture
 import CoreLocation
 
 extension LocationManager {
-
   /// The live implementation of the `LocationManager` interface. This implementation is capable of
   /// creating real `CLLocationManager` instances, listening to its delegate methods, and invoking
   /// its methods. You will typically use this when building for the simulator or device:
   ///
-  ///     let store = Store(
-  ///       initialState: AppState(),
-  ///       reducer: appReducer,
-  ///       environment: AppEnvironment(
-  ///         locationManager: LocationManager.live
-  ///       )
-  ///     )
-  ///
-  public static let live: LocationManager = { () -> LocationManager in
-    var manager = LocationManager()
-
-    manager.authorizationStatus = CLLocationManager.authorizationStatus
-
-    manager.create = { id in
+  /// ```swift
+  /// let store = Store(
+  ///   initialState: AppState(),
+  ///   reducer: appReducer,
+  ///   environment: AppEnvironment(
+  ///     locationManager: LocationManager.live
+  ///   )
+  /// )
+  /// ```
+  public static let live = Self(
```

Comment:
I'm now supplying all the arguments at once rather than assigning each endpoint. Actually found quite a few endpoints we weren't properly hooking up in the process 😬